webapp/components/Modal.vue

```diff
@@ -38,6 +39,7 @@ import ConfirmModal from '~/components/Modal/ConfirmModal'
 import DisableModal from '~/components/Modal/DisableModal'
 import ReleaseModal from '~/components/ReleaseModal/ReleaseModal.vue'
 import ReportModal from '~/components/Modal/ReportModal'
+import DeleteUserModal from '~/components/Modal/DeleteUserModal.vue'
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-03-05T09:50:31Z" title="Thursday, March 5th 2020, 10:50:31 am +01:00">Mar 5, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -30,6 +30,17 @@
       :modalData="data.modalData"
       @close="close"
     />
+    <delete-user-modal
+      v-if="open === 'delete'"
+      :id="data.id"
+      :name="data.name"
+      :slug="data.slug"
+      :avatar="data.avatar"
+      :contributionsCount="data.contributionsCount"
+      :commentedCount="data.commentedCount"
+      :createdAt="data.createdAt"
```


May add just one parameter with all the needed additional data?
In the new form with callback all these data is stored in `:modalData="data.modalData“`, see above. ☝🏼